sidestep_engine/data/preprocess_vae.py

```
@@ -51,13 +51,21 @@ def tiled_vae_encode(
-        chunk_size = TARGET_SR * 15 if gpu_mem_gb <= 8 else TARGET_SR * 30
+                chunk_size = TARGET_SR * 15 if gpu_mem_gb <= 8 else TARGET_SR * 30
+            elif torch.mps.is_available():
+                gpu_mem_gb = torch.mps.driver_allocated_memory() / (1024 ** 3)
+                chunk_size = TARGET_SR * 15 if gpu_mem_gb <= 8 else TARGET_SR * 30
+            elif torch.xpu.is_available():
+                props = torch.xpu.get_device_properties(vae_device)
+                gpu_mem_gb = props.total_mem / (1024 ** 3)
+                chunk_size = TARGET_SR * 15 if gpu_mem_gb <= 8 else TARGET_SR * 30
+            
+        except Exception:
+             pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)